packages/lesswrong/components/ea-forum/giving-portal/hooks.ts

```diff
@@ -17,12 +29,28 @@ export const useElectionCandidates = (
 
 export const useDonationOpportunities = useElectionCandidates;
 
-export const useAmountRaised = () => {
-  // TODO: Query for the actual amount
+export const useAmountRaised = (electionName: string) => {
+  const { data, loading, error } = useQuery<ElectionAmountRaisedQueryResult>(gql`
+    query ElectionAmountRaised($electionName: String!) {
+      ElectionAmountRaised(electionName: $electionName) {
+        raisedForElectionFund
+        electionFundTarget
+        totalRaised
+        totalTarget
+      }
+    }
+  `, {
+    variables: { electionName },
+  });
+
   return {
-    showAmountRaised: false,
-    raisedForElectionFund: 3720,
-    donationTarget: 10000,
-    totalRaised: 10250,
+    data: data?.ElectionAmountRaised ?? {
+      raisedForElectionFund: 0,
+      electionFundTarget: 0,
+      totalRaised: 0,
+      totalTarget: 0,
+    },
```

Comment:
Returning
```
{
      raisedForElectionFund: 0,
      electionFundTarget: 0,
      totalRaised: 0,
      totalTarget: 0,
    }
```
as a fallback is just for the convenience of having the type be non-nullable, it should never actually display these